cogito/protocol.go

```diff
@@ -212,18 +216,38 @@ func (src *Source) UnmarshalJSON(data []byte) error {
 // Validate verifies the Source configuration and applies defaults.
 func (src *Source) Validate() error {
 	//
-	// Validate mandatory fields.
+	// Evaluate mandatory fields.
```

Comment:
As stated before, `Validate()` is dynamic: some fields will become mandatory/optional based on the requested sinks.